modules/etc/image_compress_etc.cpp

```diff
@@ -166,13 +166,12 @@ static void _compress_etc(Image *p_img, float p_lossy_quality, bool force_etc1_f
 	int encoding_time = 0;
 	float effort = 0.0; //default, reasonable time
 
-	if (p_lossy_quality > 0.75) {
-		effort = 0.4;
-	} else if (p_lossy_quality > 0.85) {
-		effort = 0.6;
-	} else if (p_lossy_quality > 0.95) {
-		effort = 0.8;
-	}
+	if (p_lossy_quality > 0.95)
```

Comment:
Please add the braces back, it's part of our style guide for 4.0+.

Reply:
Done.